pyqt5ac.py

```diff
@@ -141,11 +143,16 @@ def main(rccOptions='', uicOptions='', force=False, config='', ioPaths=()):
             uicOptions = configData.get('uic_options', uicOptions)
             force = configData.get('force', force)
             ioPaths = configData.get('ioPaths', ioPaths)
+            variables = configData.get('variables', variables)
 
     # Loop through the list of io paths
     for sourceFileExpr, destFileExpr in ioPaths:
         foundItem = False
 
+        # Replace instances of the variables with the actual values of the available variables
+        for variable_name, variable_value in variables.items():
+            sourceFileExpr = sourceFileExpr.replace("%%{}%%".format(variable_name), variable_value)
```

Comment:
the same is used below except with destFileName. Would you mind extract this variable replacement into a separate function? Hence all the variable replacements would be in 1 place.